demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -1018,3 +1021,54 @@ def test_IsValidVersionValidator_fix():
         == "Updated the content item version to -1."
     )
     assert content_item.version == -1
+
+
+ERROR_MSG_TEMPLATE = "The content item name is {} , which contains an excluded word."
+
+
+@pytest.mark.parametrize(
+    "content_items, expected_number_of_failures, expected_error_message",
+    [
+        # a correct integration display name
+        ([create_integration_object()], 0, ""),
+        # an incorrect integration display name
+        (
+            [create_integration_object(paths=["display"], values=["partner"])],
+            1,
+            ERROR_MSG_TEMPLATE.format("partner"),
+        ),
+        # a correct playbook display name
+        ([create_playbook_object()], 0, ""),
+        # an incorrect playbook display name
+        (
+            [create_playbook_object(paths=["name"], values=["community"])],
+            1,
+            ERROR_MSG_TEMPLATE.format("community"),
+        ),
+        # a correct script display name
+        ([create_script_object()], 0, ""),
+        # an incorrect script display name
+        (
+            [create_script_object(paths=["name"], values=["community"])],
+            1,
+            ERROR_MSG_TEMPLATE.format("community"),
+        ),
+        # a correct classifier display name
+        ([create_classifier_object()], 0, ""),
+        # an incorrect classifier display name
+        (
+            [create_classifier_object(paths=["name"], values=["partner"])],
+            1,
+            ERROR_MSG_TEMPLATE.format("partner"),
+        ),
+    ],
+)
+def test_IsEntityNameContainExcludedWordValidator(
+    content_items, expected_number_of_failures, expected_error_message
+):
+    results = IsEntityNameContainExcludedWordValidator().is_valid(
+        content_items=content_items
+    )
+    assert len(results) == expected_number_of_failures
```

Comment:
why check multiple each time, rather than a single one in each parametrized case?

Reply:
I check only one case at every time but the results are a list of ValidationResult

Reply:
My bad. Since the expected is always 0, let's check `not len(results)` instead